### PR TITLE
feat: adds metrics counters to the js wrapper of the attestor

### PR DESCRIPTION
**Pull request by Polybius93**
This PR introduces metrics counters to the javascript layer of the attestor.

### CODE REVIEW

**Comment by scolear** [comment]
Cool! Can you add a uniform prefix to all of these metrics?

File: attestor/observer/src/config/prom-metrics.models.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      'Number of attestations created successfully'
+    ),
+    createAttestationErrorCounter: createAttestorMetricsCounter(
+      'create_attestation_error_counter',
+      'Number of errors when creating attestations'
+    ),
+  };
+}
+
+const createBlockchainObserverMetricsCounter = (network: string, name: string, help: string, version?: string) => {
```
Comment (scolear) [inline attestor/observer/src/config/prom-metrics.models.ts:59]: you can update this type from string to PrefixedChain

**Review by scolear — APPROVED** [review]

File: attestor/observer/src/config/prom-metrics.models.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      network,
+      'close_dlc_event_counter',
+      'Number of close dlc events received',
+      version
+    ),
+    postCloseDLCEventCounter: createBlockchainObserverMetricsCounter(
+      network,
+      'post_close_dlc_event_counter',
+      'Number of post close dlc events received',
+      version
```
Comment (scolear) [inline attestor/observer/src/config/prom-metrics.models.ts:96]: not all of them have the ` ?? '1' ` part unfortunately

**Review by scolear — CHANGES_REQUESTED** [review]

File: attestor/observer/src/config/prom-metrics.models.ts (hunk 2 above)
Comment (Polybius93) [reply attestor/observer/src/config/prom-metrics.models.ts:96]: Thank you, changing it!

**Pull request merged**